pkg/registry/extensions/thirdpartyresourcedata/codec_test.go

```diff
@@ -289,5 +290,40 @@ func TestThirdPartyResourceDataListEncoding(t *testing.T) {
 	if targetOutput.APIVersion != gv.String() {
 		t.Errorf("apiversion mismatch %v != %v", targetOutput.APIVersion, gv.String())
 	}
+}
+
+func TestDecodeNumbers(t *testing.T) {
+	gv := schema.GroupVersion{Group: "stable.foo.faz", Version: "v1"}
+	gvk := gv.WithKind("Foo")
+	e := &thirdPartyResourceDataEncoder{delegate: testapi.Extensions.Codec(), gvk: gvk}
+	d := &thirdPartyResourceDataDecoder{kind: "Foo", delegate: testapi.Extensions.Codec()}
+
+	// Use highest int64 number.
+	subject := &extensions.ThirdPartyResourceDataList{
+		Items: []extensions.ThirdPartyResourceData{
+			extensions.ThirdPartyResourceData{
+				Data: json.RawMessage(`{"num": 9223372036854775807}`),
```

Comment:
it's only a type-alias for `[]byte`, mandatory in `ThirdPartyResourceDataList`. `ThirdPartyResourceData` uses the plain `[]byte` type.